### DISCUSSION TITLE
Girilen Kayıtların tekillestirilmesi

### DISCUSSION THREAD

**Discussion by serdarde** [acikkaynak/deprem-yardim-projesi#53] [Feedback]
Kontrol ettigim bir cok kayit birden cokkez girilmis, bunlarin bir backoffice'de yetkili kisiler tarafindan tekillestirilmesi gerekiyor.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu çalışmamız devam etmektedir.

**Reply by serdarde** [reply]
Süper, elinize sağlık👍👍
